README.md

```diff
@@ -94,6 +97,44 @@ Once installed, the GEM will remain persistent across system reloads within the
 
 See [General Documentation](#general-documentation) for information on Guestshell and OAC.
 
+### <a name="setup-agent-auth">Puppet Agent Authentication<a>
+  
+Puppet makes use of the nxos `admin` user by default for all types in this module.  If a different user is required for puppet agent runs then the following procedure can be used to override `admin` with the desired user.
+
+**NOTE:** The user you select must already be configured on your device with the role `network-admin`.
+
+First create a different user with the role `network-admin`.
+
+~~~
+config term
+  username puppetuser password puppet role network-admin
+end
+~~~
+
+Next create a file called `cisco_node_utils.yaml` under the `cisco-network-puppet-module` files directory and add a cookie `puppetuser:local` under the `default:` yaml key.

```

Comment:
I am a bit confused by the comment here. It says, create file under 'cisco-network-puppet-module' directory but it is in  ciscopuppet/files directory. Not exactly sure how to correlate these two. Is it assumed that 'cisco-network-puppet-module' directory means '/etc/puppetlabs/code/environments/production/modules/ciscopuppet/files' ?

Reply:
I will update the comment to clarify.  I can see how it would be confusing.